lib/upgrade_info.dart

```diff
@@ -0,0 +1,43 @@
+/// This class holds the information used when upgrading from another sku.
+/// To be used with purchaseProduct and purchasePackage.
+@Deprecated('Use GoogleProductChangeInfo')
+class UpgradeInfo {
```

Comment:
This file should be added to `object_wrappers.dart`. Otherwise it's not exported